### ISSUE TITLE
Changes should be universal, instead of having to be manually configured per save file

### ISSUE THREAD

**Issue by CharlesProfessional** [Exempt-Medic/CharmChanger#7]
Changes made should not be tied to individual save files, and instead be global. Having to rebalance the charms each time you want to start a new game is a hassle.

If this is impossible for technical reasons, a good workaround would be to use a config.ini file which the user would edit, and the mod would then set its settings according to the .ini file.

**Comment by FlinGuSPluNkuS** [comment]
Yeah, It's a huge hassle to go through and change everything for each save file, making any change from the main menu global would be amazing. Thanks in advance!

**Comment by DamageMaximo** [comment]
Bad idea.

**Comment by gaahinalover** [comment]
this is exactly what I came here to mention. The fact this mod doesn't have a Global Settings config file reallllllly sucks, as I frequently start over to test new mods it really sucks having to reset ALLLLLL those settings every time, cause there's a lot I adjust
